### PR TITLE
first fixes for part 4 of threads

### PR DESCRIPTION
**Pull request by digitalist**

### CODE REVIEW

File: Threads/4_ThreadArguments/article.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
-<p>A thread gets its data per copy or per reference. Per default, you should use per copy. Why? In case your thread gets its data per reference, you have to be extremely careful about the lifetime of the arguments.</p>
```
Comment (RainerGrimm) [inline Threads/4_ThreadArguments/article.html:1]: Prefer the simpler version.

File: Threads/4_ThreadArguments/article.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+<p>A thread gets its data (arguments) by value (copying value) or by reference. By default, you should pass by value. Why? In case your thread gets its data by reference, you have to be extremely careful about the lifetime of the arguments.</p>
 <hr id="system-readmore" />
 <p>&nbsp;</p>
 <h2>Thread arguments</h2>
 <p>A thread is a <a href="https://en.wikipedia.org/wiki/Variadic_template">variadic template</a>. So it can get an arbitrary number of arguments.</p>
-<p>But now to the difference between getting the argument per copy or per reference.</p>
```
Comment (RainerGrimm) [inline Threads/4_ThreadArguments/article.html:6]: Prefer this version.

**Comment by digitalist** [comment]
you can merge this any way you want, but please, change per copy / per reference to by value/by reference)

File: Threads/4_ThreadArguments/article.html (hunk 1 above)
Comment (digitalist) [inline Threads/4_ThreadArguments/article.html:1]: you want to simplify this, or remove this line?

**Pull request merged**